usecases/objects/batch_delete.go

```diff
@@ -46,6 +46,28 @@ func (b *BatchManager) DeleteObjects(ctx context.Context, principal *models.Prin
 	return b.deleteObjects(ctx, principal, match, dryRun, output, repl, tenant)
 }
 
+// DeleteObjectsFromGRPC deletes objects in batch based on the match filter
+func (b *BatchManager) DeleteObjectsFromGRPC(ctx context.Context, principal *models.Principal,
+	params BatchDeleteParams,
+	repl *additional.ReplicationProperties, tenant string,
+) (BatchDeleteResult, error) {
+	err := b.authorizer.Authorize(principal, "delete", "batch/objects")
+	if err != nil {
+		return BatchDeleteResult{}, err
```

Comment:
Can we wrap the error here to log the function it was called in?

Reply:
I don't want to add it to every call :/

the call stacks are sometimes many many levels deep and adding too much redundant info drowns out the relevant parts.

Reply:
I've found that having the errors wrapped greatly aids when debugging as you can string search for where the error was thrown in the message. But you're right that the messages can get too long for many calls